eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -117,31 +111,37 @@ stages:
                   os: linux
 
                 templateContext:
-                  outputParentDirectory: '$(Pipeline.Workspace)'
-                  outputs:
-                    - output: nuget
-                      displayName: 'Publish ${{artifact.name}} package to NuGet.org'
```

Comment:
I'm a little bit weirded out by the output artifacts used here, as historically I believe they were added to reduce the amount of generated YML. We should try a build of `storage` or `comunication` release (just up to triggering the release) to ensure that we don't hit max yml size without having this output inlined in the templatecontext.

Reply:
The output artifacts allow you to compress your injected SDL steps by combining all of the scans across all of your outputs. This is in contract to inline output tasks like `1ES.PublishNuget` or `1ES.PublishArtifact` that will inject all of the sdl steps per inlined task.

Reply:
The amount of YML doesn't really change.  We were declaring 2 job outputs per artifact, and now we're declaring 2 publish tasks per artifact.